commands/create-account.js

```diff
@@ -48,10 +48,23 @@ async function createAccount(options) {
         keyPair = await KeyPair.fromRandom('ed25519');
         publicKey = keyPair.getPublicKey();
     }
-    await near.createAccount(options.accountId, publicKey);
     if (keyPair) {
         await near.connection.signer.keyStore.setKey(options.networkId, options.accountId, keyPair);
     }
+    try {
+        await near.createAccount(options.accountId, publicKey);
+    } catch(error) {
+        if (error.message.includes('Timeout')) {
```

Comment:
some food for thought for future work: how can we extend this so that we handle timeouts better across the board?

Reply:
I believe this is deep in the chain, and the solution is to handle it there.

Reply:
Looks like after near-api-js fix we would get `RetriesExceeded` error:

https://github.com/near/near-api-js/blob/04f7c2de6e59545f6832ade0f40f608a5c7d41e9/src/account.ts#L132